frontend/packages/ceph-storage-plugin/src/selectors/index.ts

```diff
@@ -52,22 +44,19 @@ export const getCephPVCs = (
 };
 
 export const getCephNodes = (nodesData: K8sResourceKind[] = []): K8sResourceKind[] =>
-  nodesData.filter((node) => _.keys(_.get(node, 'metadata.labels')).includes(cephStorageLabel));
+  nodesData.filter((node) => _.keys(node?.metadata?.labels).includes(cephStorageLabel));
 
-export const getCephSC = (scData: K8sResourceKind[]): K8sResourceKind[] =>
+export const getCephSC = (scData: StorageClassResourceKind[]): K8sResourceKind[] =>
   scData.filter((sc) => {
     return cephStorageProvisioners.some((provisioner: string) =>
-      _.get(sc, 'provisioner', '').includes(provisioner),
+      (sc?.provisioner ?? '').includes(provisioner),
     );
   });
 
 export const getOCSVersion = (items: FirehoseResult): string => {
```

Comment:
```suggestion
export const getOCSVersion = (items: FirehoseResult = []): string => {
```

Reply:
@Jayashree-panda you can add default values.
The idea was to provide the hint how we should be tackling default values.
It will make the functions more robust and we can avoid checking for null or defined values.
And its a good time to catch the issues with the functions. Like here we dont need a complete FireHoseResult.
We can just pass the `data` key of `FirehoseResult`.

We want to refactor and find such issues.